### DISCUSSION TITLE
Is line wrapping supported?

### DISCUSSION THREAD

**Discussion by alextricity25** [nvim-pack/nvim-spectre#204] [Q&A]
Does Spectre support line wrapping search results in the buffer?

<img width="861" alt="image" src="https://github.com/nvim-pack/nvim-spectre/assets/4964964/7fa5426c-6b55-4b32-bd99-2aa361a73cec">

**Comment by std-enigma** [comment]
Unfortunately wrapping is set to false by the plugin meaning that you can't even use autocmds to automatically set it to true on open so you have to manually set the wrapping to true every time you open it

**Comment by std-enigma** [comment]
I can easily fix this by opening a PR but there is 2 ways to go about it 

1. In the config table add a wrap flag to use that instead of the default false
2. Don't set it at all so that your global configuration determines whether line wrapping should be enabled or not

i think both are pretty reasonable but the second solution to me is much cleaner which also allows you to set the line wrapping value automatically on open by using autocmd 
but the problem is that not every user knows how to create a autocmd that's why i can't decide so let's discuss the approach

@windwp @EpsilonKu
